frontend/packages/pipelines-plugin/src/components/taskruns/list-page/TaskRunsListPage.tsx

```diff
@@ -16,19 +16,20 @@ const TaskRunsListPage: React.FC<Omit<
   React.ComponentProps<typeof ListPage>,
   'canCreate' | 'kind' | 'ListComponent' | 'rowFilters'
 > &
-  TaskRunsListPageProps> = ({ hideBadge, showPipelineColumn = true, ...props }) => {
+  TaskRunsListPageProps> = ({ hideBadge, showPipelineColumn = true, namespace, selector }) => {
```

Comment:
Don't we need to accept all other props and pass them down to `ListPage`

Reply:
Correct me if I got this wrong. As per all the props available to `TaskRunListPage`, `{...props}` only has one prop `selector`. After the changes, I see that there's no use keeping ...props for only one prop `selector` so I destructured it,

Reply:
updated and brought back ...props
![Screenshot from 2021-05-12 21-55-01](https://user-images.githubusercontent.com/24852534/118010610-d701ee80-b36c-11eb-995c-75d86d6ec1f4.png)